### PR TITLE
Update getting-started

### PR DESCRIPTION
**Pull request by MichaelClifford**
Addresses #48  

Updated the "getting-staretd.md" file to be more streamlined and reference only current content. Also broke out the section on "boards and issues" as its own doc linked here.

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -1,100 +1,29 @@
...
+*  Create a public git repository using our project template. Feel free to use the template directly or initialize your project via the cookiecutter tool.
+    * [template repo](https://github.com/aicoe-aiops/project-template)
+    * [cookiecutter](https://github.com/aicoe-aiops/cookiecutter-data-science) 
+* Start a github project board to track tasks and share ideas. Find more info and examples [here](/boards-and-issues.md)   
 
-3. Put your project document into a subfolder of this [shared directory][4]. 
 
-### **Create Git Repository from a Template**
+## Set Up Bots and CI Tools
+* Set up the aicoe-ci tools for your repo. These tools can aid in various aspects of your development workflow by running pre-commit checks, build checks, triggering pipelines to build images and more automatically. Find more info on setting up these tools [here](https://github.com/aicoe/aicoe-ci#getting-started).
```
Comment (oindrillac) [inline docs/getting-started.md:17]:
```suggestion
* Set up the [aicoe-ci](/thoth-tools.md) tools for your repo. These tools can aid in various aspects of your development workflow by running pre-commit checks, build checks, triggering pipelines to build images and more automatically. Find more info on setting up these tools [here](https://github.com/aicoe/aicoe-ci#getting-started).
```
shall we link this doc for more context?

File: docs/getting-started.md [hunk 2]
```diff
@@ -1,100 +1,29 @@
...
-
-### **Develop on Operate First Jupyterhub**
-
-The following resources are available for use to develop your project on the Operate First JupyterHub instance:
-
-* **Start developing on the Operate First JupyterHub environment**
-    - View the [video tutorial](https://www.youtube.com/watch?v=iI_-lqi3vP4&list=PL8VBRDTElCWpneB4dBu4u1kHElZVWfAwW&index=3)
-    - Access the public [JupyterHub instance][11] on the Massachusetts Open Cloud (MOC) by selecting *operate-first* authentication and then signing in with your GitHub account
-
-* **How to Monitor JupyterHub workloads using Grafana dashboards**
```
Comment (oindrillac) [inline docs/getting-started.md:53]: this seems like an important piece of info. should be a part of https://github.com/aicoe-aiops/data-science-workflows/pull/82

File: docs/getting-started.md [hunk 3]
```diff
@@ -1,100 +1,29 @@
...
-[5]: https://drivendata.github.io/cookiecutter-data-science/#cookiecutter-data-science
-[6]: https://github.com/aicoe-aiops/project-template
-[7]: https://help.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template
-[8]: https://github.com/orgs/aicoe-aiops/projects/2
-[10]: how-to-contribute.md
-[11]: https://jupyterhub-opf-jupyterhub.apps.smaug.na.operate-first.cloud/
-[12]: https://www.operate-first.cloud/hitchhikers-guide/apps/docs/odh/jupyterhub/runbook.md
-[13]: https://grafana.operate-first.cloud/
+* Attend our [meetups](https://www.operate-first.cloud/data-science/operate-first-data-science-community/docs/meetup-landing-page.md).
+* Say "Hi" on [Slack](https://operatefirst.slack.com/ssb/redirect).
```
Comment (oindrillac) [inline docs/getting-started.md:28]: on the #data-science channel? :)

**Review by oindrillac — COMMENTED** [review]
looks good!
/lgtm

File: docs/getting-started.md (hunk 2 above)
Comment (MichaelClifford) [reply docs/getting-started.md:53]: yes. I agree. it can be moved to a different doc

**Pull request merged**